### PR TITLE
Set error level psalm to 1.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: src/PDO/SchemaPDOOracle.php [hunk 1]
```diff
@@ -165,7 +166,9 @@ protected function loadTableSchema(string $name): ?TableSchema
      */
     protected function loadTablePrimaryKey(string $tableName): ?Constraint
     {
-        return $this->loadTableConstraints($tableName, 'primaryKey');
+        /** @var mixed */
+        $tablePrimaryKey = $this->loadTableConstraints($tableName, 'primaryKey');
```
Comment (darkdef) [inline src/PDO/SchemaPDOOracle.php:170]: need use constant

File: src/PDO/SchemaPDOOracle.php [hunk 2]
```diff
@@ -177,7 +180,9 @@ protected function loadTablePrimaryKey(string $tableName): ?Constraint
      */
     protected function loadTableForeignKeys(string $tableName): array
     {
-        return $this->loadTableConstraints($tableName, 'foreignKeys');
+        /** @var mixed */
+        $tableForeingKeys = $this->loadTableConstraints($tableName, 'foreignKeys');
```
Comment (darkdef) [inline src/PDO/SchemaPDOOracle.php:184]: need use constant

File: src/PDO/SchemaPDOOracle.php [hunk 3]
```diff
@@ -245,7 +250,9 @@ protected function loadTableIndexes(string $tableName): array
      */
     protected function loadTableUniques(string $tableName): array
     {
-        return $this->loadTableConstraints($tableName, 'uniques');
+        /** @var mixed */
+        $tableUniques = $this->loadTableConstraints($tableName, 'uniques');
```
Comment (darkdef) [inline src/PDO/SchemaPDOOracle.php:254]: constant

File: src/PDO/SchemaPDOOracle.php [hunk 4]
```diff
@@ -257,7 +264,9 @@ protected function loadTableUniques(string $tableName): array
      */
     protected function loadTableChecks(string $tableName): array
     {
-        return $this->loadTableConstraints($tableName, 'checks');
+        /** @var mixed */
+        $tableCheck = $this->loadTableConstraints($tableName, 'checks');
```
Comment (darkdef) [inline src/PDO/SchemaPDOOracle.php:268]: constant

File: src/PDO/SchemaPDOOracle.php [hunk 5]
```diff
@@ -427,7 +439,7 @@ public function getLastInsertID(string $sequenceName = ''): string
             /* get the last insert id from the master connection */
             $sequenceName = $this->quoteSimpleTableName($sequenceName);
 
-            return $this->db->useMaster(function (ConnectionPDOInterface $db) use ($sequenceName) {
+            return (string) $this->db->useMaster(static function (ConnectionPDOInterface $db) use ($sequenceName) {
```
Comment (darkdef) [inline src/PDO/SchemaPDOOracle.php:442]: static function(ConnectionPDOInterface $db, $sequenceName)  ?

File: src/PDO/SchemaPDOOracle.php (hunk 1 above)
Comment (terabytesoftw) [reply src/PDO/SchemaPDOOracle.php]: done.

File: src/PDO/SchemaPDOOracle.php (hunk 2 above)
Comment (terabytesoftw) [reply src/PDO/SchemaPDOOracle.php]: done.

File: src/PDO/SchemaPDOOracle.php (hunk 3 above)
Comment (terabytesoftw) [reply src/PDO/SchemaPDOOracle.php]: done.

File: src/PDO/SchemaPDOOracle.php (hunk 4 above)
Comment (terabytesoftw) [reply src/PDO/SchemaPDOOracle.php]: done.

File: src/PDO/SchemaPDOOracle.php (hunk 5 above)
Comment (terabytesoftw) [reply src/PDO/SchemaPDOOracle.php:457]: I thought of that myself, but the callback has only one argument: https://github.com/yiisoft/db/blob/976e42595ebfa08a7064d64cdbd957712f0b5269/src/Connection/Connection.php#L209

**Pull request merged**